cmd/boulder-va/main.go

```diff
@@ -27,7 +27,8 @@ type Config struct {
 		// before giving up. May be short-circuited by deadlines. A zero value
 		// will be turned into 1.
 		DNSTries                  int
-		DNSProvider               *cmd.DNSProvider `validate:"required"`
+		DNSProvider               *cmd.DNSProvider `validate:"required_without=DNSStaticResolvers"`
+		DNSStaticResolvers        []string         `validate:"required_without=DNSProvider"`
```

Comment:
This field should be commented, I've thrown something to together below:

```suggestion
		// Resolvers is a list of DNS resolvers. Each entry must include a
		// host/IP and port separated by a colon. Additionally, if the host is a
		// literal IPv6 address, it must be enclosed in square brackets.
		DNSStaticResolvers        []string         `validate:"required_without=DNSProvider"`
```

Reply:
thanks!  added